File: C_Sharp_code/Example/WindowsFormsApp/VB_demo01/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim u As String = TextBox1.Text
        Dim p As String = TextBox2.Text
        MessageBox.Show("帳號:" + u + vbCrLf + "密碼:" + p, "系統訊息", MessageBoxButtons.YesNo, MessageBoxIcon.Asterisk)
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Dim s As String = Interaction.InputBox("輸入字串", "輸入視窗")
        MessageBox.Show(s, "系統訊息", MessageBoxButtons.YesNo, MessageBoxIcon.Asterisk)
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo01/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using Microsoft.VisualBasic;

namespace CS_demo01
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            string u = textBox1.Text;
            string p = textBox2.Text;
            MessageBox.Show("帳號:" + u + "\r\n密碼:" + p, "系統訊息", MessageBoxButtons.YesNo, MessageBoxIcon.Asterisk);

        }

        private void button2_Click(object sender, EventArgs e)
        {
            string s;
            s = Interaction.InputBox("輸入字串", "輸入視窗");//參考VB類別庫
            MessageBox.Show(s, "系統訊息", MessageBoxButtons.YesNo, MessageBoxIcon.Asterisk);
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            Form2 form2 = new Form2();
            form2.Show();
            //this.close();
        }
    }
}
